Tests/PubNubTests/Integration/SubscriptionIntegrationTests.swift

```diff
@@ -98,17 +103,22 @@ class SubscriptionIntegrationTests: XCTestCase {
           XCTAssertFalse(pubnub.subscribedChannels.contains(self.testChannel))
           unsubscribeExpect.fulfill()
         }
+      case .messageReceived:
+//        print("messageReceived: \(message)")
+        pubnub.unsubscribe(from: [self.testChannel])
+        publishExpect.fulfill()
       case let .connectionStatusChanged(status):
-        print("Status: \(status)")
+//        print("connectionStatusChanged: \(status)")
```

Comment:
Do we leave these commented out lines in for future/later usage?

Reply:
Yeah it's just a convenience for testing.